content/batch/artifact.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Configure Artifact Repository"
+date: 2018-11-18T00:00:00-05:00
+weight: 50
+draft: false
+---
+
+### Configure Artifact Repository
+
+Argo uses an artifact repository to pass data between jobs in a workflow, known as artifacts. Amazon S3 can be used as an artifact repository.
+
+Let's create a S3 bucket using the AWS Service Operator from earlier.
```

Comment:
I think instructions are missing for creating S3 bucket using AWS Service Operator?

Reply:
The instructions are there, as well as a hidden section underneath for the AWS CLI commands.